release notes for 6.8.3

pcsanwald opened the pull request:
6.8.3 release notes.

elasticmachine commented:
Pinging @elastic/es-docs

lcawl commented on docs/reference/release-notes/6.8.asciidoc:
```diff
@@ -1,3 +1,78 @@
+* Use system context for looking up connected nodes {pull}43991[#43991] (issue: {issue}43974[#43974])
+
+Machine Learning::
+* [DOCS] Adds allow no datafeeds query param to the GET, GET stats and STOP datafeed APIs (7.2-6.1) {pull}44864[#44864]
```
This one should be omitted. I've added the >docs label to the PR

lcawl commented on docs/reference/release-notes/6.8.asciidoc:
```diff
@@ -1,3 +1,75 @@
+* Include leases in error message when operations no longer available {pull}45681[#45681]
+
+Infra/Core::
+* Add OCI annotations and adjust existing annotations {pull}45167[#45167] (issues: {issue}45162[#45162], {issue}45166[#45166])
```
I think 45162 and 45166 are related PRs, not issues.

lcawl commented on docs/reference/release-notes/6.8.asciidoc:
```diff
@@ -1,3 +1,75 @@
+* Ensure AsyncTask#isScheduled remain false after close {pull}45687[#45687] (issue: {issue}45576[#45576])
+
+Engine::
+* Handle delete document level failures {pull}46100[#46100] (issue: {issue}46083[#46083])
```
Likewise, I think 46083 is a related PR, not an issue fixed by 46100

lcawl commented on docs/reference/release-notes/6.8.asciidoc:
```diff
@@ -1,3 +1,75 @@
+* Handle no-op document level failures {pull}46083[#46083]
+
+Features/Indices APIs::
+* Check shard limit after applying index templates {pull}44619[#44619] (issues: {issue}34021[#34021], {issue}44567[#44567], {issue}44619[#44619])
```
Likewise, I think 34021 is just a related PR not a fixed issue.
The 44619 also appears twice here--once as a PR and once as an issue.

debadair replied:
It looks like the script is treating all "Relates" and "Supersedes" references as issues. (The duplicate entry is because the Supersedes reference is referencing the same PR.)

lcawl commented on docs/reference/release-notes/6.8.asciidoc:
```diff
@@ -1,3 +1,75 @@
+* Add rest_total_hits_as_int in HLRC's search requests {pull}46076[#46076] (issue: {issue}43925[#43925])
+
+Infra/Core::
+* Add default CLI JVM options {pull}44545[#44545] (issues: {issue}219[#219], {issue}42021[#42021])
```
It seems to me that #219 is unrelated

debadair replied:
It was a reference to an issue in elasticsearch-docker.

nik9000 commented:
@elasticmachine, run elasticsearch-ci/docs

pcsanwald commented:
@lcawl @debadair thanks for your contributions and review, I was quite ill on Friday so that's why I was radio silent. Will review and update according to your comments.

pcsanwald commented:
@lcawl thanks for your review, I believe I've addressed your comments, mind having another look? 

I think you and @debadair are right that release notes script is indeed including "relates" PRs as issues, I'll look into this.
